src/js/simon/renderSimon.ts

```diff
@@ -0,0 +1,66 @@
+import { getById } from '../utilities';
+
+export function renderSimon(): void {
+
+	const colours = [["Red", "red"], ["Blue", "blue"], ["Green", "green"], ["Yellow", "yellow"]];
```

Comment:
We should export this from utilities because another module uses it. I'm thinking it could be all objectey. Something like this
```
export const Colours = {
Red: { Lower: "red", Title: "Red"},
Blue: {Lower: "blue", Title: "Blue"},
};
```
Or, even cooler, it could be a class with a `AsLower` property so we only have to write the words once.

Reply:
Yeah I haven't done classes with typescript yet. From what I've seen, they are a bit confusing.